pilot/pkg/model/sidecar.go

```diff
@@ -168,9 +175,7 @@ func DefaultSidecarScopeForNamespace(ps *PushContext, configNamespace string) *S
 	defaultEgressListener.virtualServices = ps.VirtualServicesForGateway(&dummyNode, constants.IstioMeshGateway)
 
 	out := &SidecarScope{
-		Config: &config.Config{Meta: config.Meta{
-			Namespace: configNamespace,
-		}},
+		Namespace:          configNamespace,
```

Comment:
Can we add a "default-sidecar" or some thing for consistency?

Reply:
SG